Fast vis source subtraction

surajit4444mondal opened the pull request:
Changes made to support #82

peijin94 commented:
The PR looks all good to me.

minor comment: 
in image_ms(), the variable "delete" is not used
in source_substraction.py the elif in Line 230 and Line 222 are mutually exclusive to each other, so line 230 can be replaced by an 'else'.

surajit4444mondal commented:
Thank you Peijin for the suggestions. The variable "delete" should be used to used to delete the model as is done in image_ms_quick. I have added those lines to image_ms. I also added delete_allsky and sky_image variables to image_ms. I agree with you that the lines you pointed in source_subtraction can be replaced with "else". I have now changed the code accordingly.
